taxonomy-editor-frontend/src/pages/project/editentry/index.tsx

```diff
@@ -45,13 +62,22 @@ const EditEntry = ({ taxonomyName, branchName, id }: EditEntryProps) => {
           <Typography sx={{ mb: 2, mt: 2, ml: 2 }} variant="h4">
             You are now editing &quot;{id}&quot;
           </Typography>
-          <IconButton
-            sx={{ ml: 1, color: greyHexCode }}
-            onClick={() => setOpenDeleteDialog(true)}
-          >
-            <DeleteOutlineIcon />
-          </IconButton>
+          {!isExternalNode && (
+            <IconButton
+              sx={{ ml: 1, color: greyHexCode }}
+              onClick={() => setOpenDeleteDialog(true)}
+            >
+              <DeleteOutlineIcon />
+            </IconButton>
+          )}
         </Stack>
+        {isExternalNode && (
+          <CustomAlert
+            message="This node has been imported from another taxonomy to extend the current taxonomy. You can only add children from the current taxonomy to it."
```

Comment:
Indicating were the nodes comes from would be great. See my comment about changing is_external to another property style.

Reply:
suggestion (probably to do a separate PR): if you keep track of the file SHA and with the `src_position`, you can even give a link to the Github file on the precise file (although it may not be true for huge taxonomies)